docs/core/rules.rst

```diff
@@ -0,0 +1,113 @@
+:desc: TBD.
+
+.. _rules:
+
+Rules
+=====
+
+.. edit-link::
+
+.. contents::
+   :local:
+
+Rasa rules are a form of training data used to train Rasa's dialogue management models.
+Rules describe parts of conversations which should always follow the same path.
+Useful applications for rules are for example:
+
+* FAQs: FAQs are questions which users ask out of context. Rules are an easy way to
+    specify fixed answers to the questions.
+* Fallbacks: Users might confront your assistant with messages which your assistant
+    can't handle yet. Unless these messages are close to existing training data, Rasa
+    NLU will typically show low intent confidence levels for these messages. You can
+    use :ref:`fallback-classifier` to treat low NLU confidence like an FAQ
+* Forms (TODO: LINK): It's a common use case for assistants to collect form-like data
+    from the user. Both, activation of forms as well as handling of unexpected
+    events as part of a form, are usually following fixed paths.
+
+    (OTHER ideas?)
+
+Do we need a rules vs. stories section?
```

Comment:
Rules vs. stories would definitely be helpful to clarify when to use one vs. the other and how they take precedent